packages/viewer/package.json

```diff
@@ -1,77 +1,82 @@
 {
-  "name": "@speckle/viewer",
-  "version": "2.4.2",
-  "description": "A 3d viewer for Speckle, based on threejs.",
-  "homepage": "https://speckle.systems",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/specklesystems/speckle-server.git",
-    "directory": "packages/viewer"
-  },
-  "main": "dist/speckleviewer.js",
-  "module": "dist/speckleviewer.esm.js",
-  "sourceType": "module",
-  "files": [
-    "dist"
-  ],
-  "engines": {
-    "node": "^16.0.0"
-  },
-  "scripts": {
-    "build": "NODE_ENV=production rollup --config",
-    "build:dev": "rollup --config",
-    "dev": "rollup --config --watch",
-    "example": "EXAMPLE_BUILD=1 rollup --config && http-server ./example -p 3032 -o",
-    "prepublishOnly": "npm run build",
-    "lint": "eslint . --ext .js,.ts"
-  },
-  "author": "AEC Systems",
-  "license": "Apache-2.0",
-  "keywords": [
-    "speckle",
-    "aec",
-    "3d",
-    "3d viewer",
-    "threejs"
-  ],
-  "dependencies": {
-    "@speckle/objectloader": "^2.4.2",
-    "camera-controls": "^1.33.1",
-    "core-js": "^3.21.1",
-    "hold-event": "^0.1.0",
-    "lodash.debounce": "^4.0.8",
-    "rainbowvis.js": "^1.0.1",
-    "regenerator-runtime": "^0.13.7",
-    "three": "^0.136.0"
-  },
-  "devDependencies": {
-    "@babel/cli": "7.15.7",
-    "@babel/core": "7.15.8",
-    "@babel/eslint-parser": "^7.15.8",
-    "@babel/plugin-proposal-class-properties": "^7.14.5",
-    "@babel/plugin-proposal-private-methods": "^7.14.5",
-    "@babel/plugin-transform-classes": "^7.16.0",
-    "@babel/preset-env": "7.15.8",
-    "@babel/preset-react": "7.14.5",
-    "@babel/preset-typescript": "7.15.0",
-    "@rollup/plugin-babel": "^5.3.1",
-    "@rollup/plugin-commonjs": "^21.0.3",
-    "@rollup/plugin-node-resolve": "^13.1.3",
-    "babel-jest": "27.2.5",
-    "babel-loader": "^8.0.0-beta.4",
-    "babel-plugin-add-module-exports": "1.0.4",
-    "babel-plugin-transform-class-properties": "6.24.1",
-    "browserslist": "^4.20.2",
-    "eslint": "^8.11.0",
-    "eslint-config-prettier": "^8.5.0",
-    "http-server": "^14.1.0",
-    "jest": "27.2.5",
-    "mocha": "^9.1.2",
-    "prettier": "^2.5.1",
-    "rollup": "^2.70.1",
-    "rollup-plugin-delete": "^2.0.0",
-    "rollup-plugin-terser": "^7.0.2",
-    "yargs": "^17.2.1"
-  },
-  "gitHead": "5627e490f9a3ecadf19cc4686ad15f344d9ad2d3"
+    "name": "@speckle/viewer",
```

Comment:
Because of the formatting config changes you added (the prettier config file) it's now reformatted the entire file, please undo the changes and re-do the ones you did without the reformatting